spring-boot-starter-camunda/src/main/resources/application-camunda-saas.yaml

```diff
@@ -9,6 +9,7 @@ camunda:
       enabled: true
       audience: zeebe.camunda.io
       base-url: https://${camunda.client.cluster-id}.${camunda.client.region}.zeebe.camunda.io
+      rest-address: https://${camunda.client.cluster-id}.${camunda.client.region}.zeebe.camunda.io
```

Comment:
This would mean that grpc and rest are served from the same url in saas. Is that true?